lib/features/guide_view/guide_view.dart

```diff
@@ -19,11 +22,15 @@ import "widgets/guide_grid.dart";
 import "widgets/guide_tile.dart";
 
 @RoutePage()
-class GuideView extends StatelessWidget {
+class GuideView extends HookWidget {
   const GuideView({super.key});
 
   @override
   Widget build(BuildContext context) {
+    useEffectOnInit(() {
+      Future.microtask(requestFCMPermission);
```

Comment:
What's the purpose of this code here? (why here)

Reply:
Yeah, I know this seems funny, but I just didn't want to show it as first things you enter the app and thought that once you enter the guide tab seems a good moment to ask for permissions. Just random idea. But we can change that if you have a better idea where to place it

Reply:
Yeah, I know this seems funny, but I just didn't want to show it as first things you enter the app and thought that once you enter the guide tab seems a good moment to ask for permissions. Just random idea. But we can change that if you have a better idea where to place it